libweb3jsonrpc/UnixSocketServer.cpp

```diff
@@ -92,6 +124,10 @@ bool UnixDomainSocketServer::StartListening()
 
 bool UnixDomainSocketServer::StopListening()
 {
+	int dummy = 0x90;
+	if (::write(m_exitChannel[1], &dummy, sizeof(dummy)) < 0)
+		cerr << "Failed to notify UNIX domain server loop. " << strerror(errno) << "\n";
```

Comment:
Better output it to `clog(VerbosityWarn, "rpc")`, It will not need `\n` in the end.